dht_test.go

```diff
@@ -767,6 +767,7 @@ func TestRefreshBelowMinRTThreshold(t *testing.T) {
 
 	// and because of the above bootstrap, A also discovers E !
 	waitForWellFormedTables(t, []*IpfsDHT{dhtA}, 4, 4, 20*time.Second)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
:nose:

Reply:
Tests are horribly flaky, and this helps to pass them. It gives more time to the RT to converge.

I will work on the tests in another PR